### PR TITLE
Ported eslint configuration from hradmin project

### PR DESCRIPTION
**Pull request by gustaff-weldon**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,48 +1,67 @@
 /**
  * Shareable ESLint-er config used in all bayzat projects.
  * @author Michal Kechner <kepek@bayzat.com>
+ * @author Bernard Lange <kepek@bayzat.com>
```
Comment (pleszkowicz) [inline index.js]: We like so much kepek@bayzat.com, that we all have the same email :)

Comment (kepek) [inline index.js]: :D

Comment (gustaff-weldon) [inline index.js]: Lol :)

**Comment by kepek** [comment]
@gustaff-weldon when you're planing to merge this? I would need to block SIR to the current one

**Pull request merged**